microsoft-edge/devtools-guide-chromium/performance/reference.md

```diff
@@ -0,0 +1,1238 @@
+
+![The Memory checkbox](./reference-images/memory-highlight.png)
+
+DevTools displays a **Memory** chart, above the **Summary** tab.  There is also a chart below the **NET** chart, called **HEAP**.  The **HEAP** chart provides the same information as the **JS Heap** line in the **Memory** chart:
+
+![Memory metrics](./reference-images/memory-chart.png)
+
+The colored lines on the chart map to the colored checkboxes above the chart.  Clear a checkbox to hide that category from the chart.
+
+The chart only displays the region of the recording that is currently selected.  For example, in the previous figure, the **Memory** chart is only showing memory usage from around the 3600 ms mark to the 6200 ms mark.
+
+See also:
+* [Visualize memory leaks (Performance tool: Memory checkbox)](#visualize-memory-leaks-performance-tool-memory-checkbox), below.
+
+
+<!-- ------------------------------ -->
+#### View the duration of a portion of a recording
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#duration -->
+
+When analyzing a section like **Network** or **Main**, sometimes you need a more precise estimate of how long certain events took.  Hold **Shift**, click and hold, and drag left or right to select a portion of the recording.  At the bottom of your selection, DevTools shows how long that portion took:
+
+![Viewing the duration of a portion of a recording](./reference-images/main-duration.png)
+
+
+<!-- ------------------------------ -->
+#### View a screenshot
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#view-screenshot -->
+
+To learn how to turn on screenshots, see [Capture screenshots while recording](#capture-screenshots-while-recording), above.
+
+To view a screenshot of how the page looked during that moment of the recording, hover over the **Overview**.  The **Overview** is the section that contains the **CPU** and **NET** charts:
+
+![Viewing a screenshot](./reference-images/screenshots-hover.png)
+
+You can also view screenshots by selecting a frame in the **Frames** section.  DevTools displays a small version of the screenshot in the **Summary** tab:
+
+![Viewing a screenshot in the Summary tab](./reference-images/summary-preview.png)
+
+To zoom in on the screenshot, click the thumbnail in the **Summary** tab.
+
+
+<!-- ------------------------------ -->
+#### View layers information
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#layers -->
+
+To view advanced layers information about a frame:
+
+1. In the **Performance** tool, click the **Capture settings** (![Capture settings](./reference-images/capture-settings-icon.png)) button, and then select the **Enable advanced rendering instrumentation (slow)** checkbox.
+
+1. In the **Frames** section, select a frame.  DevTools displays information about the layers in the **Layers** tab at the bottom of the **Performance** tool:
+
+   ![The Layers pane](./reference-images/layers-all.png)
```

Comment:
This image is also broken.

Reply:
@captainbrosset  pls try:

1. go to internal preview anchor:
https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/performance/reference?branch=pr-en-us-3378#view-layers-information
2. In Edge, **F12** (DevTools) > Right-click **Refresh** > **Empty cache and hard refresh** > **F12**.

Predict the pngs are rendered.
 
Expect that to work, b/c the pngs are rendered at the below, and it is a frequent behavior/ action required.

external preview anchor:
https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/perf-sync/microsoft-edge/devtools-guide-chromium/performance/reference.md#view-layers-information